playbooks/roles/atomic_registry/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: 'install the docker package'
+  yum:
+    name: docker
+    state: latest
+
+- name: 'install the atomic package'
+  yum:
+    name: atomic
+    state: latest
+
+- name: 'install Atomic Registry docker image'
+  docker_image:
+    name: "{{ atomic_registry_upstream_image }}"
+
+- name: 'install master script'
+  shell: atomic install "{{ atomic_registry_upstream_image_name }}" executable=/usr/bin/bash
+  environment:
+    REGISTRYIMAGE: "{{ atomic_registry_registry_image }}"
+    REGISTRYTAG: "{{ atomic_registry_registry_tag }}"
+    REGISTRYPORT: "{{ atomic_registry_registry_port }}"
+    CONSOLEIMAGE: "{{ atomic_registry_console_image }}"
+    CONSOLETAG: "{{ atomic_registry_console_tag }}"
+    CONSOLEPORT: "{{ atomic_registry_console_port }}"
+    MASTERIMAGE: "{{ atomic_registry_master_image }}"
+    MASTERTAG: "{{ atomic_registry_master_tag }}"
+    MASTERPORT: "{{ atomic_registry_master_port }}"
+
+- name: 'enable Aatomic Registry services'
+  service: name=atomic-registry-master.service enabled=yes state=started
+
+- name: 'delay 15 seconds moment'
+  shell: sleep 15
```

Comment:
It can take a short while for all three docker containers to start; they must be running before continuing. In addition I have encountered a case where one of the containers would start and then quickly die in which case the first check would succeed when it should really fail; that is the reason for the initial 15sec pause before the checking starts.